packages/fuels/package.json

```diff
@@ -22,12 +22,20 @@
       "require": "./dist/cli.js",
       "import": "./dist/cli.mjs",
       "types": "./dist/cli.d.ts"
+    },
+    "./test-utils": {
+      "require": "./dist/test-utils.js",
+      "import": "./dist/test-utils.mjs",
+      "types": "./dist/test-utils.d.ts"
```

Comment:
Why are we adding this to the umbrella package?

I was under the impression that all test utilities should be imported from `@fuel-ts/utils`